src/panel/poi_bloc/opening_minimal.js

```diff
@@ -13,6 +13,7 @@ export default class MinimalHourPanel {
   async set(poi) {
     let openingBlock = poi.blocks.find((block) => block.type === 'opening_hours')
     this.opening = null
+    this.isTwentyFourSeven = openingBlock.is_24_7
```

Comment:
`openinBlock` may be null here, causing unhandled errors.

Reply:
i move it to the test and init it in the constructor